Cyotek.Windows.Forms.ImageBox/ImageBox.cs

```diff
@@ -4062,21 +4098,63 @@ protected override void OnMouseWheel(MouseEventArgs e)
     {
       base.OnMouseWheel(e);
 
-      if (this.AllowZoom && this.SizeMode == ImageBoxSizeMode.Normal)
```

Comment:
Perhaps an old fashioned opinion, but I'm not a fan of local functions, can you please promote this to be the garden variety private method.

Reply:
Sure, no big thing. Thats really just a thing of the style. Since it's your project, I am going to align this.

Reply:
Thanks for following my whims. I should probably just use them, but it always feels like both a code smell and also a means of making a method harder to read.